### PR TITLE
Introduce new bundle.proto

### PR DESCRIPTION
**Pull request by wu-hui**
#no-changelog

### CODE REVIEW

File: Firestore/Protos/nanopb/firestore/bundle.nanopb.cc [hunk 1]
```diff
@@ -0,0 +1,217 @@
+/*
+ * Copyright 2018 Google
```
Comment (wilhuff) [inline Firestore/Protos/nanopb/firestore/bundle.nanopb.cc:2]: This should be `Copyright 2021 Google LLC`.

`build_protos.py` is the thing that's setting this but it currently adds the copyright unconditionally.

File: Firestore/Protos/protos/firestore/bundle.proto [hunk 2]
```diff
@@ -0,0 +1,121 @@
+// Copyright 2020 Google LLC.
```
Comment (wilhuff) [inline Firestore/Protos/protos/firestore/bundle.proto:1]: 2021

Also, go/copyright does not include a trailing period.

File: Firestore/Protos/protos/firestore/bundle.proto [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+  // The number of documents in the bundle.
+  uint32 total_documents = 4;
+
+  // The size of the bundle in bytes, excluding this `BundleMetadata`.
+  uint64 total_bytes = 5;
+}
+
+// A Firestore bundle is a length-prefixed stream of JSON representations of
+// `BundleElement`.
+// Only one `BundleMetadata` is expected, and it should be the first element.
```
Comment (wilhuff) [inline Firestore/Protos/protos/firestore/bundle.proto:107]: Is this meant to be a continuation of the previous paragraph or a new paragraph? If the former, please join with the previous line and reflow the text. If the latter, please add an additional blank line between paragraphs.

**Review by wilhuff — APPROVED** [review]
LGTM

File: Firestore/Protos/nanopb/firestore/bundle.nanopb.cc (hunk 1 above)
Comment (wu-hui) [reply Firestore/Protos/nanopb/firestore/bundle.nanopb.cc]: Fixed.

**Pull request merged**